lib/apiservers/engine/backends/vicbackends.go

```diff
@@ -114,3 +118,40 @@ func ProductVersion() string {
 func VchConfig() *config.VirtualContainerHostConfigSpec {
 	return vchConfig
 }
+
+func pingPortLayer() {
+	params := misc.NewPingParamsWithContext(context.TODO())
+	ticker := time.NewTicker(RetryTimeSeconds * time.Second)
+	defer ticker.Stop()
```

Comment:
You may want to add an `Infof` message here that we're waiting for the PL to come up.  Otherwise the log will be empty and we won't know what the deamon is doing.